ci/phpunit/run-split-tests.sh

```
@@ -0,0 +1,75 @@
+PROJECT_ROOT="$(cd "${SCRIPT_DIR}/../.." && pwd)"
+
+# Default test directories to scan
+TEST_DIRS=(
+  "tests/MartinGeorgiev/Doctrine/DBAL/Types"
+  "tests/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions"
+  "tests/MartinGeorgiev/Utils"
+)
+
+# Parse command line arguments
+COVERAGE_ARGS=""
+if [[ "$@" == *"--coverage-clover"* ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)